crates/db_views/src/comment_view.rs

```diff
@@ -190,21 +190,6 @@ fn queries<'a>() -> Queries<
       query = query.filter(comment_like::score.eq(-1));
     }
 
-    let is_creator = options.creator_id == options.local_user.map(|l| l.person.id);
```

Comment:
Good. I'm sure these just got copy-pasted from the `post_view` ones, where it does make sense to have them.